vault/dendron.ref.config.publishing.md

```diff
@@ -0,0 +1,138 @@
+---
+id: q960Z622o0RLO32w0B8SP
+title: Publishing
```

Comment:
it seems redundant to have both this and [[Config|dendron.topic.publish.config]]. can we consolidate by explicitly enumerating children in [[Configuration|dendron.ref.config]] and have the children links point to `dendron.topic.*.config`?

Reply:
> it seems redundant to have both this and [[Config|dendron.topic.publish.config]]. 

agreed

> can we consolidate by explicitly enumerating children in [[Configuration|dendron.ref.config]] and have the children links point to dendron.topic.*.config?

can you elaborate? I don't understand what you mean by that.